roles/common/meta/main.yml

```diff
@@ -10,10 +10,13 @@ dependencies:
       - repo: 'deb {{ apt_repos.sensu.repo }} sensu main'
         key_url: '{{ apt_repos.sensu.key_url }}'
         validate_certs: no
+    when: ansible_architecture != "ppc64le"
```

Comment:
I don't think we want to take in code that won't support running sensu on those systems.